### PR TITLE
Create GET endpoint for rooms

### PR DESCRIPTION
**Pull request by alankbi**
### List of Changes:

* Create GET endpoint for checking if a room exists
* Add relevant DTO classes, methods, and tests

### Notes:

* Takes in the roomId as a query parameter instead of as part of the request body
* If no room is found, returns the standard 404 room not found message
* If a room is found, returns an object with a single roomId variable

### CODE REVIEW

**Comment by alankbi** [comment]
[Create GET endpoint for rooms](https://trello.com/c/T6qiHD1T/98-create-get-endpoint-for-rooms)

File: src/test/java/com/rocketden/main/RoomTests.java [hunk 1]
```diff
@@ -86,6 +118,26 @@ public void createValidRoom() throws Exception {
         CreateRoomResponse actual = Utility.toObject(jsonResponse, CreateRoomResponse.class);
 
         assertEquals(expected.getHost(), actual.getHost());
+
+        // Send GET request to validate that room exists
+
```
Comment (zpChris) [inline src/test/java/com/rocketden/main/RoomTests.java:123]: nit: Remove this extra line.

**Review by zpChris — APPROVED** [review]
Just left two nits; otherwise, this looks great!

**Review by helenli8181 — APPROVED** [review]
This looks good to me!

**Pull request merged**